src/Graphql/Type/SchemaBuilder.php

```diff
@@ -146,7 +144,7 @@ private function getResourceFieldConfiguration(string $fieldDescription = null,
         try {
             $graphqlType = $this->convertType($type, $isInput, $isMutation, $mutationName);
             $graphqlWrappedType = $graphqlType instanceof WrappingType ? $graphqlType->getWrappedType() : $graphqlType;
-            $isInternalGraphqlType = in_array($graphqlWrappedType, GraphQLType::getInternalTypes(), true);
+            $isInternalGraphqlType = \in_array($graphqlWrappedType, GraphQLType::getInternalTypes(), true);
```

Comment:
`\` not needed

Reply:
It is, it trigger an opcode optimization (PHP EA Inspection reports it).

Reply:
Okay but we discussed it before and if we add it here shouldn't we put this everywhere? Oh okay so we actually came to the decision that we will haha nvm.

Reply:
Why only for the `in_array` function then?

Reply:
https://github.com/Roave/FunctionFQNReplacer